### DISCUSSION TITLE
Date and Time on Client

### DISCUSSION THREAD

**Discussion by seancarte** [ltsp/ltsp#231] [Q&A]
I have the correct time on the server, which is running ntp:
root@ltsp:~# timedatectl status
                      Local time: Fri 2020-01-17 08:48:02 SAST
                  Universal time: Fri 2020-01-17 06:48:02 UTC
                        RTC time: Fri 2020-01-17 06:48:02
                       Time zone: Africa/Johannesburg (SAST, +0200)
       System clock synchronized: yes
systemd-timesyncd.service active: yes
                 RTC in local TZ: no

But on the client the time is offset by two hours:
![image](https://user-images.githubusercontent.com/18610043/72590519-9627d000-3906-11ea-924f-683731803ba3.png)

The client can't reach the NTP server; its only access to the Internet is through the squid proxy server running on the LTSP server.

Is there a way to fix this? The only way I can think of is to run an NTP server on the LTSP server. But then on the LTSP server, I'd have to configure the NTP client to connect to itself. Is there a better solution?

**Comment by alkisg** [comment]
So if I understood this correctly, this question isn't LTSP-specific, right?

"How can I sync PC time if it it can only access the Internet via a proxy server"?

If that's the question, then wouldn't it be best to ask it on a more popular site like stackexchange etc?

Example answer: https://superuser.com/questions/307158/how-to-use-ntpdate-behind-a-proxy

I think htpdate sounds good...

**Comment by seancarte** [comment]
Thanks, @alkisg , I'll look into that.

But the LTSP-related question is: why does the time on the client jump forward by two hours? Presumably that's because my timezone is GMT+2. Is there no way to prevent the LTSP client doing that?

**Comment by alkisg** [comment]
There's no code for time management in LTSP at all.
Whatever you see, isn't caused by LTSP.

Are the clients dual boot? If so, Windows puts the clock in local time, while Linux uses UTC.

**Comment by seancarte** [comment]
Thanks yet again, @alkisg ! The client is a dual-boot laptop. I tried with a PC and the time is not affected.
